dist/rei-dot-com/scss/cdr-tokens.scss

```diff
@@ -2284,6 +2284,144 @@ $cdr-membership-subtle-color-border-button-primary-rest: #13352c;
 $cdr-membership-subtle-color-border-button-primary-active: #13352c;
 $cdr-membership-subtle-color-border-button-primary-hover: #13352c;
 
+// @supports statements can be removed once Safari 15 is no longer supported
+
+// XS
+@mixin cdr-xs-cq-up($name: '') {
```

Comment:
Couple questions 
Am I correct in thinking that if $name is not provided, #{$name} will be an empty string, will this be problematic?

Reply:
No, this is ok and suppored. Providing a container name to a container query is optional. When it's not provided here, it just ends up being an extra space.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries#naming_containment_contexts